src/lib.js

```diff
@@ -838,6 +849,14 @@ var program,
             else {
               request.body.mode = 'raw';
               request.body.raw = rawDataString;
+
+              request.body.options = {};
+              request.body.options.raw = {};
+              request.body.options.raw.language = Object.keys(LANGUAGE_REGEX_MATCH)
+                .find((key) => {
+                  return LANGUAGE_REGEX_MATCH[key].test(content_type);
```

Comment:
Can we wrap this around try catch? This is to make sure regex doesn't break entire import in case of bad input.

Reply:
Makes sense. I have made the change.